components/post_view/post_body_additional_content/post_body_additional_content.tsx

```diff
@@ -149,22 +149,26 @@ export default class PostBodyAdditionalContent extends React.PureComponent<Props
         );
     }
 
+    renderAppBindings = () => {
+        return (
+            <EmbeddedBindings
+                embeds={this.props.post.props.app_bindings}
+                post={this.props.post}
+            />
+        );
+    }
```

Comment:
```suggestion
    renderAppBindings = () => (
        <EmbeddedBindings
            embeds={this.props.post.props.app_bindings}
            post={this.props.post}
        />
    );
```

Reply:
Leaving it as-is to match other functions in this file